docs/src/modules/components/AppTableOfContents.js

```diff
@@ -2,18 +2,18 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import throttle from 'lodash/throttle';
-import { styled, alpha } from '@mui/material/styles';
+import { styled } from '@mui/material/styles';
 import Typography from '@mui/material/Typography';
 import Link from 'docs/src/modules/components/Link';
 import PageContext from 'docs/src/modules/components/PageContext';
 import { useTranslate } from 'docs/src/modules/utils/i18n';
 
 const Nav = styled('nav')(({ theme }) => {
```

Comment:
Something feels off with the text alignment on the TOCs:

<img width="208" alt="Screenshot 2021-12-20 at 15 31 16" src="https://user-images.githubusercontent.com/3165635/146832619-3aabfaac-c30c-4aeb-8508-6774f9e92294.png">

Should the "Contents" has a padding-left of 11px over 12px? Currently, it feels a bit out of alignment.

Reply:
Thanks for pointing out!

Reply:
Tackled in #30371.